connector/MongoDBAtlasODBC.pq

```diff
@@ -36,6 +30,13 @@ MongoDBAtlasODBCType = type function (
             Documentation.SampleValues = {"db"}
         ]
     ),
+    optional loglevel as (
+        type text meta [
+            Documentation.FieldCaption = "Logging Level",
+            Documentation.FieldDescription = "Logging level for the MongoDB Atlas SQL ODBC Driver",

```

Comment:
```suggestion
            Documentation.FieldDescription = "Logging level for the MongoDB Atlas SQL ODBC Driver. Default value: INFO",
```
It would be helpful to specify that the Log Level is INFO if it isn't specified.